reana_job_controller/config.py

```diff
@@ -19,9 +21,14 @@
 """Root path of the shared volume ."""
 
 JOB_BACKENDS = {
-    'kubernetes': KubernetesJobManager
+    'kubernetes': KubernetesJobManager,
+    'cern_htcondor': CernHTCondorJobManager
 }
 """Supported job backends and corresponding management class."""
 
 DEFAULT_JOB_BACKEND = 'kubernetes'
 """Default compute backend for job submission."""
+
+HTCONDOR_DOCKER_IMG = os.getenv('HTCONDOR_DOCKER_IMG',
```

Comment:
Let's use a more verbose and descriptive name as in R-Workflow-Controller, see [here](https://github.com/reanahub/reana-workflow-controller/blob/2709032255454bc0c9be85067d2b29eb75c1efe2/reana_workflow_controller/config.py#L73) for example.